tests/test_core.py

```diff
@@ -0,0 +1,28 @@
+import pytest
```

Comment:
pytest is unused here

Reply:
I left there on purpose. It's just an habit to import pytest on test files. If you mind that much I can remove it